packages/site/mosaic.config.js

```diff
@@ -7,7 +7,7 @@ const siteConfig = {
     ...mosaicConfig.plugins,
     {
       modulePath: '@jpmorganchase/mosaic-plugins/SidebarPlugin',
-      options: { rootDirGlob: '*/*' }
+      options: { rootDirGlob: '*' }
```

Comment:
this flips us back to a single sidebar for everything..
I think we were going to converge with Salt.. is this a change we still want ?

Reply:
There is no sidebar without this...

Reply:
removed this code...it was only for testing